crml/eth-bridge/src/tests.rs

```diff
@@ -33,6 +36,35 @@ use sp_core::{
 	Public, H256,
 };
 use sp_runtime::offchain::StorageKind;
+use sp_runtime::SaturatedConversion;
+
+fn create_latest_block_mock(block_number: u64, block_hash: H256, timestamp: U256) -> EthBlock {
```

Comment:
nit: does it guarantee whatever block given will become the "latest"?
if not I'd call it `mock_block_response` and can doc comment the behaviour about the highest block number becoming latest

otherwise `LatestOrNumber` should  be given instead of `block_number` but that's probably be overkill